package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "./bin.js example.js --open -p 8080 -A 0.0.0.0",
+    "build": "NODE_ENV=production ./bin.js build example.js --optimize",
```

Comment:
Not sure, but wouldn't it be even more handy that one could expect that `--optimize` does that without a need to specify `NODE_ENV=production`?

I prefer explicit over implicit though, but in this use case I'd expect from `--optimize` to do the right thing without a need to specify (and for that matter we'd also need to document it) NODE_ENV. Or is this too magical? I'm not really sure 😊